egs/wsj_mix/asr1/local/mix_2_spk_max_tt_mix

```diff
@@ -0,0 +1,333 @@
+440c0401_-0.6154_442c0412_0.6154
```

Comment:
Maybe it's better to move this file to Google Drive.

Reply:
OK. I'll do it.

Reply:
I'm fine to keep them as they are because the numbers of likes are at most 30K lines. Another possibility is to include the compressed files and uncompress them when needed. Can you revert them?